tools/firely-auth/firely-auth-management-app.rst

```diff
@@ -0,0 +1,100 @@
+.. _firely_auth_mgmt:
+
+Firely Auth User Management
+===========================
+
+When using the SQL Server user store, you can manage the users therein with a command line utility called the 'Firely Auth Management app'. This can be used both interactively from the commandline and automated from a script or devops pipeline.
+
+.. _firely_auth_mgmt_exe:
+
+Executable
+----------
+
+The app is included in the :ref:`binaries <firely_auth_deploy_exe>` of Firely Auth.
+You can invoke it with:
+
+.. code:: Powershell
+
+    ./Firely.IdentityServer.ManagementApp.exe
```

Comment:
Here and below:
`.\Firely.IdentityServer.ManagementApp.exe` for windows
and
`dotnet ./Firely.IdentityServer.ManagementApp.exe` for MacOS/Linux

Reply:
See above: `/` works in Windows as  well.